src/authz/schemaRegistry.test.ts

```diff
@@ -1,30 +1,30 @@
 import * as assert from "assert";
-import sinon from "sinon";
+import sinon, { SinonStubbedInstance } from "sinon";
```

Comment:
```suggestion
import * as sinon from "sinon";
```
Minor change that should allow just using `sinon.SinonStubbedInstance`, unless we wanted to pivot to only using named imports